app/javascript/Domain/ImperialGameCoordinator.js

```diff
@@ -117,6 +121,10 @@ export default class ImperialGameCoordinator {
       return;
     }
 
+    if (action.type !== 'undo') {
+      this.#undoHistory.addUndoCheckpoint();
```

Comment:
This will add a possible undo to every action, right? The way undo currently works is that it will undo an entire turn at a time and not just one action at a time. Are we changing behavior with this?

Reply:
Ah, good catch, let me fix that (not sure why it didn't fail for the games on my local machine)

Reply:
[1c3054e](https://github.com/Thrillberg/imperial/pull/745/commits/1c3054eee2299941ea44ad0d96cb6f5389718f08)

Moved into `setOldState` which is also named `#addUndoCheckpoint` now